libs/wordpress-plugin/plugin/trunk/block/edit-or-preview/edit.tsx

```diff
@@ -130,20 +174,34 @@ export const Edit = ({
       return;
     }
 
-    const { data: subgraph } = await getEntitySubgraph({
+    const { data: subgraph, errors } = await getEntitySubgraph({
       data: { entityId, graphResolveDepths: blockSubgraphResolveDepths },
     });
 
     if (!subgraph) {
-      throw new Error("No data returned from getEntitySubgraph");
+      displayToast({
+        message:
+          errors?.[0]?.message ||
+          `Error retrieving Block Protocol entity with id '${entityId}'`,
+        actions: [
+          {
+            label: "Get Help",
+            url: "https://blockprotocol.org/contact",
+          },
+        ],
+      });
+      return;
     }
 
     setEntitySubgraph(subgraph);
-  }, [entityId]);
+  }, [displayToast, entityId]);
 
   const serviceCallbacks = useMemo<ServiceEmbedderMessageCallbacks>(
-    () => constructServiceModuleCallbacks(),
-    [],
+    () =>
+      constructServiceModuleCallbacks((toastProps) =>
+        displayToast(toastProps, { toastId: "billing" }),
```

Comment:
This seems to be the only place where we explicitly specify a `toastId`. Does this mean we don't stack the errors for equivalent toasts?

Reply:
This is the only error that will be repeated in short succession – the others won't be so we don't need to deduplicate them